components/Favourites/Favourites.js

```diff
@@ -0,0 +1,144 @@
+import styled from "styled-components";
+import Carousel from "react-multi-carousel";
+import PropTypes from "prop-types";
+
+import { Divider, ItemCard } from "../../styles/UIKit";
+import { FAVOURITES_TITLE } from "../../utils/constants";
+
+const responsive = {
+  superLargeDesktop: {
+    // the naming can be any, depends on you.
+    breakpoint: { max: 4000, min: 3000 },
+    items: 5,
+  },
+  desktop: {
+    breakpoint: { max: 3000, min: 1024 },
+    items: 5.5,
+  },
+  tablet: {
+    breakpoint: { max: 1024, min: 464 },
+    items: 2,
+  },
+  mobile: {
+    breakpoint: { max: 464, min: 0 },
+    items: 1,
+  },
+};
+
+let NextOne, PreviousOne;
```

Comment:
I am not sure how will they work on multiple re-renders, Don't you think these should be state vars? you are assigning functions to them what if there definition changes on re-render??

Reply:
Ignore it is done outside the component.